src/pages/basic_data/app_admin/CustomerInstitutionMetadataForm.tsx

```diff
@@ -93,89 +110,91 @@ export const CustomerInstitutionMetadataForm = ({
+                          canAssignDoi: false,
+                          customer: {
+                            ...emptyCustomerInstitution,
+                            name,
+                            displayName: name,
+                            cristinId: selectedInstitution?.id ?? '',
+                          },
+                          doiAgent: emptyProtectedDoiAgent,
+                        });
+                        setCustomerData(selectedInstitution);
+                      }}
+                      errorMessage={touched && !!error ? error : undefined}
+                      fieldInputProps={field}
+                    />
+                  </Box>
+                )}
+              </Field>
+            )}
+            <CustomerInstitutionInformationFromCristin
+              customerData={customerData}
+              cristinId={values.customer.cristinId}
+              setName={(newName) => {
+                setFieldValue(CustomerInstitutionFieldNames.DisplayName, newName);
+                setFieldValue(CustomerInstitutionFieldNames.Name, newName);
+              }}
+              displayName={values.customer.displayName}
             />
 
-            <Field name={CustomerInstitutionFieldNames.Sector}>
-              {({ field }: FieldProps) => (
-                <div>
-                  <FormLabel component="legend" sx={{ fontWeight: 'bold' }}>
-                    {t('basic_data.institutions.sector')}
-                  </FormLabel>
-                  <Box sx={{ display: 'flex', gap: '0.5rem', mt: '0.5rem' }}>
-                    {Object.values(Sector).map((sector) => (
-                      <Chip
-                        key={sector}
-                        data-testid={dataTestId.basicData.institutionAdmin.sectorChip(sector)}
-                        label={t(`basic_data.institutions.sector_values.${sector}`)}
-                        color="primary"
-                        variant={field.value === sector ? 'filled' : 'outlined'}
-                        onClick={() => setFieldValue(field.name, sector)}
-                      />
-                    ))}
-                  </Box>
-                </div>
-              )}
-            </Field>
+            <Divider />
+
+            <Box sx={{ display: 'flex', justifyContent: 'flex-start', gap: '1.5rem' }}>
+              <Field name={CustomerInstitutionFieldNames.Sector}>
+                {({ field }: FieldProps) => (
+                  <FormControl fullWidth>
+                    <InputLabel id="demo-simple-select-label"> {t('basic_data.institutions.sector')}</InputLabel>
+                    <Select
+                      fullWidth
+                      required
+                      variant="filled"
+                      labelId="sector-select-label"
+                      id="sector-select"
+                      value={field.value}
+                      onChange={(sector) => setFieldValue(field.name, sector.target.value)}>
```

Comment:
`Select` er litt funky, og mange plasser er det faktisk enklere å bruke `<TextField select>` for å oppnå det samme:
```suggestion
                  <TextField
                    {...field}
                    select
                    fullWidth
                    required
                    label={t('basic_data.institutions.sector')}
                    variant="filled">
```